### ISSUE TITLE
Design token for inactive tab text

### ISSUE THREAD

**Issue by tyler-hoffman** [ant-design/ant-design#43946]
### What problem does this feature solve?

The tab component has design tokens for `itemActiveColor`, `itemHoverColor`, and `itemSelectedColor`, but no design token for deactivated states. `colorText` can be used to change this color, but it also affects the color of the `items` props' children.

### What does the proposed API look like?

An additional `itemColor` design token that affects tab titles when they are not in a hover/active/selected state.

**Comment by monolithed** [comment]
Totally agree, see my styles for a basic tab design:

<img width="629" alt="Screenshot 2023-08-02 at 14 39 10" src="https://github.com/ant-design/ant-design/assets/803674/c449142a-d56d-4356-8adb-daed0b66f26f">

```jsx
<AntTabs className={classes}
         renderTabBar={(props, TabNavList) => (
                  <TabNavList {...props} mobile={false} />
         )}
         popupClassName={styles.popupClassName}
         destroyInactiveTabPane={true}
         tabBarStyle={{backgroundColor: 'none'}}
         defaultActiveKey="1"
         animated={true}
         type="editable-card"
         tabBarGutter={0}
         hideAdd={true}
         items={Body(items)}
{...props}
/>
```

```ts
{
    components: {
                Tabs: {
                    colorText: 'rgba(0, 0, 0, 75%)',
                    colorPrimary: 'rgba(0, 0, 0, 85%)',
                    colorPrimaryHover: '#000',
                    colorBgContainer: 'rgba(255, 211, 54, 100%)',
                }
    }
}
```


```css
.tabs,
.tabs :global(.ant-tabs-tab-active) {
    border-radius: var(--border-radius--middle) !important;
}

.tabs :global(.ant-tabs-nav-wrap) {
    --tabs-fade-color: rgb(255, 255, 255, 83%);
}

.tabs :global(.anticon-ellipsis) {
    font-size: var(--font-size--default);
}

.tabs :global(.ant-tabs-nav-wrap::before) {
    box-shadow: inset 20px 0 8px -8px var(--tabs-fade-color) !important;
}

.tabs :global(.ant-tabs-nav-wrap::after) {
    /*box-shadow: inset -20px 0 8px -8px var(--tabs-fade-color) !important;*/
}

.tabs {
    /* https://github.com/ant-design/ant-design/issues/43363#issuecomment-1621365921 */
    /*min-height: 500px;*/
    /*height: 570px;*/
    overflow: hidden;
}

.popupClassName :global(.ant-tabs-dropdown-menu) {
    background-color: white !important;
}
```

And only overriding the box-shadow property a bit helped me to fix the rounded-borders:

<img width="196" alt="Screenshot 2023-08-02 at 14 48 50" src="https://github.com/ant-design/ant-design/assets/803674/ce5553ea-96ef-4901-9c1f-825d7dd8f7b2">

**Comment by MadCcc** [comment]
Will be released in 5.9.0
